typing/printtyp.ml

```diff
@@ -332,10 +332,12 @@ let ident ppf id = pp_print_string ppf
 (* Print a path *)
 
 let ident_stdlib = Ident.create_persistent "Stdlib"
+let opened_idents = lazy (ident_stdlib :: List.map Ident.create_persistent !Clflags.open_modules)
```

Comment:
The list `open_modules` may contain module paths (`A.B`) that are not valid persistent names.

Reply:
Thanks, indeed. Do you think the feature still makes sense if we restrict to those paths which *do* make sense?